apiserver/facades/client/application/application_unit_test.go

```diff
@@ -1948,8 +1948,8 @@ func (s *ApplicationSuite) TestAddUnits(c *gc.C) {
 	ctrl := s.setup(c)
 	defer ctrl.Finish()
 
-	unitName := "postgresql/99"
-	newUnit := s.expectUnit(ctrl, unitName)
+	unitName := coreunit.MustNewName("postgresql/99")
+	newUnit := s.expectUnit(ctrl, unitName.String())
```

Comment:
**suggestion:** I think it would be better to make expectUnit take an unitName instead of a string as second arg.

Reply:
I think that's fair. The reason I take a string is because it's mostly used to configure Mongo state mocks which return strings. We will just have to call `.String()` internally a bunch of times, rather than externally to this method, probably an improvement